distributed/http/worker/prometheus/core.py

```diff
@@ -20,52 +22,54 @@ def collect(self):
         from prometheus_client.core import GaugeMetricFamily
 
         tasks = GaugeMetricFamily(
-            "dask_worker_tasks", "Number of tasks at worker.", labels=["state"]
+            self.build_name("tasks"),
+            "Number of tasks at worker.",
+            labels=["state"],
         )
-        tasks.add_metric(["stored"], len(self.worker.data))
-        tasks.add_metric(["executing"], self.worker.executing_count)
-        tasks.add_metric(["ready"], len(self.worker.ready))
-        tasks.add_metric(["waiting"], self.worker.waiting_for_data_count)
-        tasks.add_metric(["serving"], len(self.worker._comms))
+        tasks.add_metric(["stored"], len(self.server.data))
+        tasks.add_metric(["executing"], self.server.executing_count)
+        tasks.add_metric(["ready"], len(self.server.ready))
+        tasks.add_metric(["waiting"], self.server.waiting_for_data_count)
+        tasks.add_metric(["serving"], len(self.server._comms))
```

Comment:
Since we're cleaning up the namespace and have breaking changes in here anyhow, I suggest to change this one as well. This is published as a task gauge but, in fact, it is the number of comms in the comm pool. I'm not even sure if this serves any actual value, I'd be fine with removing it. Moving it to a different gauge would also be ok for me